components/badge/badge.css

```diff
@@ -0,0 +1,53 @@
+:root {
+    --notification-badge-color: #2dc0e7;
+    --online-badge-color: #008000;
+    --offline-badge-color:#ff0000;
+    --dnd-badge-color: #808080;
+    --white-color: #ffffff;
```

Comment:
everything is great but if we set variables for colours we shouldn't name these as black/white instead we can name these based on the use case.